lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * View for an individual team.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore', 'gettext', 'teams/js/views/team_discussion',
```

Comment:
@cahrens Yes, @muzaffaryousaf is working of this branch https://github.com/edx/edx-platform/pull/9183.